tests/publishing/guest-pools.test.js

```diff
@@ -1,8 +1,95 @@
-const common = require('../common'),
+'use strict';
+const Promise = require('bluebird'),
+			common = require('../common'),
       expect = require('chai').expect;
 
 describe('Guest Pools', () => {
-  before(() => common.login(browser, 'admin', 'admin', 'local'));
-  it('should navigate to Guest Pools', () =>
-    common.clickSidebarTab(browser, 'Guest Pools'));
+  beforeEach(() => { return common.login(browser, 'admin', 'admin', 'local')
+		.then(() => common.clickSidebarTab(browser, 'Guest Pools'));
+	});
+  after(() => common.logout());
+  
+  // it('should navigate to Guest Pools', () =>
+  //   common.clickSidebarTab(browser, 'Guest Pools'));
+
+  it('should not allow adding a pool if there are no templates', () => {
+  	return browser.waitForExist('//*[@id="add_pool"]')
+  		.then(() => browser.isExisting('//*[@id="add_pool" and contains(@class,"disabled")]'))
+  		.then((ex) => {
+  			if(ex){
+					return common.waitAndClick('//*[@id="add_pool"]')
+						.then(() => browser.waitForExist('//*[@id="pool_form" and @class="hidden"]'))
+				}
+				else {
+					return console.log('Templates are available.')
+				}
+  	});
+  });
+
+  it('should allow cancellation of creating a new pool', ()=> {
+  	 return browser.waitForExist('//*[@id="add_pool"]')
+  		.then(() => browser.isExisting('//*[@id="add_pool" and contains(@class,"disabled")]'))
+  		.then((ex) => {if(!ex){ 
+  			return common.waitAndClick('//*[@id="add_pool"]')
+  				.then(() => common.waitAndClick('//*[@id="add_pool_form"]/div[8]/button[2 and text()="Cancel"]'))
+					.then(() => browser.waitForExist('//*[@id="add_pool" and not(contains(@class,"hidden"))]')); 		
+			}});
+  });
+
+  it('should allow creating a new pool when a template exists', () => {
```

Comment:
it isn't. Was just putting this in by hand. I will be moving all of this to a scenario.